IFC4x3/Properties/p/PullInVoltage_2dz9sHSqLEL9DJguaAm4sn/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PullInVoltage_2dz9sHSqLEL9DJguaAm4sn" Name="PullInVoltage" UniqueId="a7f49d91-7345-4e54-9353-ab890ac04db1" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
I would definitely propose to look into a more generic solution where the pset supplies the context.

Reply:
This is a specific term associated to relay. It is not the same with RatedVoltage or Voltage (working voltage) in IFC. Some reference here: https://www3.panasonic.biz/ac/e/control/relay/term/index.jsp